flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableSourceITCase.java

```diff
@@ -264,6 +264,146 @@ void testLimit() throws Exception {
                 .containsAll(result);
     }
 
+    @Test
+    public void testFilter() throws Exception {
```

Comment:
Could you also add a test in `JdbcTablePlanTest` like `JdbcTablePlanTest#testLimitPushDown`?

Reply:
Sure, will do